README.md

```diff
@@ -36,6 +36,7 @@ The app is translated into multiple languages:
 - Polish (FAQs missing)
 - Portuguese
 - Swedish (FAQs missing)
+- French (FAQs missing)
```

Comment:
Thanks for adding the language here. Please move it up to have the list in alphabetical order.